model/k8s/metrics.yaml

```
@@ -74,6 +74,34 @@ groups:
       The actual accuracy would depend on the instrumentation and operating system.
     instrument: gauge
     unit: "s"
+  - id: metric.k8s.node.allocatable_cpu
```

```suggestion
  - id: metric.k8s.node.allocatable.cpu
```
I wonder what others think, but I'd consider making `allocatable` also a namespace. https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/naming.md#general-naming-considerations

Sorry if I missed a past discussion about this.

---

I won't be against it, but this will require changes in the k8scluster receiver to match the new semconv